cblox/include/cblox/core/submap_collection.h

```diff
@@ -68,13 +68,13 @@ class SubmapCollection {
   // Activate a submap
   // NOTE(alexmillane): Note that creating a new submap automatically activates
   //                    it.
-  void activateSubMap(const SubmapID submap_id);
+  void activateSubmap(const SubmapID submap_id);
 
   // Interacting with the submap poses
-  bool setSubMapPose(const SubmapID submap_id, const Transformation &pose);
-  void setSubMapPoses(const TransformationVector &transforms);
-  bool getSubMapPose(const SubmapID submap_id, Transformation *pose_ptr) const;
-  void getSubMapPoses(TransformationVector* submap_poses) const;
+  bool setSubmapPose(const SubmapID submap_id, const Transformation &pose);
+  void setSubmapPoses(const TransformationVector &transforms);
+  bool getSubmapPose(const SubmapID submap_id, Transformation *pose_ptr) const;
+  void getSubmapPoses(TransformationVector *submap_poses) const;
```

Comment:
I think somehow our autoformaters are set up to place `&` and `*` symbols in different locations.

I just checked the style guide and actually both are allowed (https://google.github.io/styleguide/cppguide.html#Pointer_and_Reference_Expressions), however, our convention in the team is to have the space after the symbol, rather than before. I think it makes sense to stick with that for consistency with the rest of our code.

If you like I can just format this file.

Reply:
Ok, I'll update my autoformater to fit with the team conventions.